cube/macOS/cube/cube.cmake

```diff
@@ -28,14 +28,14 @@ set(cube_RESOURCES ${CMAKE_BINARY_DIR}/staging-json/MoltenVK_icd.json
     ${CMAKE_CURRENT_SOURCE_DIR}/macOS/cube/Resources/VulkanIcon.icns)
 
 # Have Xcode handle the Storyboard
-if(${CMAKE_GENERATOR} MATCHES "^Xcode.*")
```

Comment:
Instead of checking CMake generator for Xcode it's simpler to just use the established XCODE variable.

https://cmake.org/cmake/help/latest/variable/XCODE.html#xcode